energyplus_api_helpers/import_helper.py

```diff
@@ -37,43 +69,56 @@ class EPlusAPIHelper:
     This is the primary helper class for providing usability to E+ API clients
     This class intentionally provides strings as path outputs to keep the conversion to strings reduced in the client
     """
-    def __init__(self, eplus_install_path: Path):
-        self.eplus_install_path = eplus_install_path
+
+    def __init__(self, eplus_install_path: Optional[Path] = None):
+        if eplus_install_path is None:
+            self.eplus_install_path = _infer_energyplus_install_dir()
+            print(f"Infered Location of EnergyPlus installation at {self.eplus_install_path}")
+        else:
+            if not isinstance(eplus_install_path, Path):
+                eplus_install_path = Path(eplus_install_path)
+            if not (eplus_install_path / 'pyenergyplus').is_dir():
+                raise ValueError(f"Wrong eplus_install_path, '{eplus_install_path}/pyenergyplus' does not exist")
+            self.eplus_install_path = eplus_install_path
```

Comment:
EPlusAPIHelper:

* If not provided, infer the E+ directory.
* If provided: Handle wrong argument early too.